src/dsp_tools/models/resource.py

```diff
@@ -38,14 +38,14 @@
 class KnoraStandoffXmlEncoder(json.JSONEncoder):
     """Classes used as wrapper for DSP standoff-XML"""
 
-    def default(self, obj) -> str:
```

Comment:
the superclass defines `default(self, o)`, so it's not recommended to rename `o` to `obj`